<launch>
  <arg name="show_rviz" default="true"/>
  <!-- Tracker Nodes-->
  <node name="ibgho_tracker_node" pkg="vtec_tracker" type="ibgho_tracker_node" output="screen">
    <rosparam command="load" file="$(find vtec_tracker)/config/config.yaml"/>
  </node>

  <node name="keyboard_control" pkg="vtec_tracker" type="keyboard_control" output="screen"/>

  <group if="$(arg show_rviz)">
  	<node name="rviz" type="rviz" pkg="rviz" args="-d $(find vtec_tracker)/config/vtec_tracker.rviz"/>
  </group>

</launch>